javascript/javascript1/week2/weather.js

```diff
@@ -0,0 +1,14 @@
+function clothesBaseTemperature(temperature) {
```

Comment:
You should probably add something like "getClothesBasedOnTemperature"

Reply:
ok